code/association_scan/TensorQTL/TensorQTL.ipynb

```diff
@@ -115,29 +120,41 @@
-    "- variant_id - ID of the top variant (rsid or chr:position:ref:alt)\n",
-    "- tss_distance - Distance of the SNP to the gene transcription start site (TSS)\n",
-    "- ma_samples - Number of samples carrying the minor allele\n",
-    "- ma_count - Total number of minor alleles across individuals\n",
-    "- af - Alternative allele frequency in MiGA cohort\n",
-    "- ref_factor - Flag indicating if the alternative allele is the minor allele in the cohort (1 if AF <= 0.5, -1 if not)\n",
-    "- pval_nominal - Nominal P-value from linear regression\n",
-    "- slope - Slope of the linear regression\n",
-    "- slope_se - Standard error of the slope\n",
-    "- pval_perm - First permutation P-value directly obtained from the permutations with the direct method\n",
-    "- pval_beta - Second permutation P-value obtained via beta approximation. This is the one to use for downstream analysis"
+    "- chrom: chromosome number\n",
+    "- pos : Variant chromosomal position (basepairs)\n",
+    "- n_variant: Total number of variants tested in cis\n",
+    "- beta_shape1: First parameter value of the fitted beta distribution\n",
+    "- beta_shape2: Second parameter value of the fitted beta distribution\n",
+    "- true_df: Effective degrees of freedom the beta distribution approximation\n",
+    "- p_true_df: Empirical P-value for the beta distribution approximation\n",
+    "- variant_id: ID of the top variant (rsid or chr:position:ref:alt)\n",
+    "- tss_distance: Distance of the SNP to the gene transcription start site (TSS)\n",
+    "- tes_distance: Distance of the SNP to the gene transcription end site (TES)\n",
+    "- ma_samples: Number of samples carrying the minor allele\n",
+    "- ma_count: Total number of minor alleles across individuals\n",
+    "- af: Alternative allele frequency\n",
+    "- p_nominal: Nominal P-value from linear regression\n",
+    "- bhat: Slope of the linear regression\n",
+    "- sehat: Standard error of the slope\n",
+    "- p_perm: First permutation P-value directly obtained from the permutations with the direct method\n",
+    "- p_beta: Second permutation P-value obtained via beta approximation. This is the one to use for downstream analysis\n",
+    "- molecular_trait_object_id: Molecular trait identifier.(gene)\n",
+    "- n_traits: group size in the permutation test\n",
+    "- genomic_inflation: genomic inflation factor(lambda), quantifying the extent of the bulk inflation and the excess false positive rate. \n",
+    "mutiple testing corrected results:\n",
+    "- q_beta: q value for p_beta using Storey's method(qvalue), more conservative than FDR\n",
+    "- q_perm: q value for p_perm using Storey's method(qvalue), more conservative than FDR\n",
+    "- fdr_beta: adjusted p value for p_beta using Benjamini-Hochberg method(FDR)\n",
+    "- fdr_perm: adjusted p value for p_beta using Benjamini-Hochberg method(FDR)\n",
+    "- p_nominal_threshold: the threshold value derived from the Beta distribution\n",
```

Comment:
shoud be "Nominal p-value threshold for variants in the corresponding molecular trait, derived from empirical beta distribution as a result of permutation testing"